### PR TITLE
Add LexiLoader.app v30

### PR DESCRIPTION
**Pull request by sailavu**

### CODE REVIEW

File: Casks/lexiloader.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'lexiloader' do
+  version '30'
+  sha256 '68c0d9482fe8f3b8c7251b452929ba4b58bbfdf478f5dc68740edc09fd87d860'
+
+  url "https://dl.dropboxusercontent.com/u/6682148/LexiLoader_v#{version}.zip"
```
Comment (vitorgalvao) [inline Casks/lexiloader.rb]: Needs a `# dropboxusercontent.com is the official download host per the vendor homepage` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/cb487ad405f74cb1f26b93e3f3e8412c82a22e34/Casks/0ad.rb#L5)).

Comment (sailavu) [inline Casks/lexiloader.rb]: Added the line and fixed whitespace too.

**Comment by vitorgalvao** [comment]
Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by sailavu** [comment]
Done. Thanks for pointing it out. Learnt something new.

**Pull request merged**